src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -56,30 +65,198 @@ public function getLockOwner(): ?string
+            /** @var null|object{description: string, geo?: string, osm?: string} $object */
+            $object = $this->data['location'];
+            return $object;
+        }
+
+        $location = null;
+
+        $nodeList = $this->xpath->query($this->getXpathPrefix() . '/podcast:location');
+
+        if ($nodeList->length > 0) {
+            /** @var DOMElement $item */
+            $item                  = $nodeList->item(0);
+            $location              = new stdClass();
+            $location->description = $item->nodeValue;
+            $location->geo         = $item->getAttribute('geo');
+            $location->osm         = $item->getAttribute('osm');
+        }
+
+        $this->data['location'] = $location;
+
+        return $this->data['location'];
+    }
+
+    /**
+     * Get the podcast images
+     *
+     * @return null|object{srcset: string}
+     */
+    public function getPodcastIndexImages(): object|null
+    {
+        if (array_key_exists('images', $this->data)) {
+            /** @var null|object{srcset: string} $object */
+            $object = $this->data['images'];
+            return $object;
+        }
+
+        $images = null;
+
+        $nodeList = $this->xpath->query($this->getXpathPrefix() . '/podcast:images');
+
+        if ($nodeList->length > 0) {
+            /** @var DOMElement $item */
+            $item           = $nodeList->item(0);
+            $images         = new stdClass();
+            $images->srcset = $item->getAttribute('srcset');
+        }
+
+        $this->data['images'] = $images;
+
+        return $this->data['images'];
+    }
+
+    /**
+     * Get the podcast update frequency
+     *
+     * @return null|object{description: string, complete?: bool, dtstart?: string, rrule?: string}
+     */
+    public function getPodcastIndexUpdateFrequency(): object|null
+    {
+        if (array_key_exists('updateFrequency', $this->data)) {
+            /** @var null|object{description: string, complete?: bool, dtstart?: string, rrule?: string} $object */
+            $object = $this->data['updateFrequency'];
+            return $object;
+        }
+
+        $updateFrequency = null;
+
+        $nodeList = $this->xpath->query($this->getXpathPrefix() . '/podcast:updateFrequency');
+
+        if ($nodeList->length > 0) {
+            /** @var DOMElement $item */
+            $item                         = $nodeList->item(0);
+            $updateFrequency              = new stdClass();
+            $updateFrequency->description = $item->nodeValue;
+            $updateFrequency->complete    = $item->getAttribute('complete');
+            $updateFrequency->dtstart     = $item->getAttribute('dtstart');
+            $updateFrequency->rrule       = $item->getAttribute('rrule');
+        }
+
+        $this->data['updateFrequency'] = $updateFrequency;
+
+        return $this->data['updateFrequency'];
+    }
+
+    /**
+     * Get the podcast people
+     *
+     * @return list<stdClass|null>
```

Comment:
```suggestion
     * @return list<object|null>
```

If you add a class-level doc block of:

```php
/**
 * @psalm-type PersonType = object{
 *     name: string,
 *     role: string,
 *     other-prop: string|null|etc…,
 * }
 */
```

Then, you can annotate this method with `@return list<PersonType>|null` and use `PersonType` anywhere relevant: https://psalm.dev/docs/annotating_code/supported_annotations/#psalm-type